src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
```

Comment:
Let's use more descriptive names for the variables. What is obj?
Try to specify what this variable is